src/library/searchqueryparser.cpp

```diff
@@ -180,14 +191,20 @@ void SearchQueryParser::parseTokens(QStringList tokens,
                 // For untagged strings we search the track fields as well
                 // as the crate names the track is in. This allows the user
                 // to use crates like tags
-                std::unique_ptr<OrNode> gNode = std::make_unique<OrNode>();
+                qDebug() << searchColumns;
+                if (searchColumns.contains("crate")) {
```

Comment:
Would this ever be false? If not, please remove the `if` statement.

Reply:
yes. the test cases use this a lot. Also, I'm thinking about the library rewrite branch and some additions I'm planning for the search query there. As I wrote, I did not want to break the API which means that you specify the columns your unnamed query should be matched against. Now we just have special matches as well and those need to be handled.

Reply:
okay